webapi/src/main/scala/org/knora/webapi/responders/Responder.scala

```diff
@@ -48,15 +48,25 @@ object Responder {
 
 }
 
+/**
+  * Data needed to be passed to each responder.
+  *
+  * @param system the actor system.
+  * @param applicationStateActor the application state actor ActorRef.
+  * @param responderManager the responder manager ActorRef.
+  * @param storeManager the store manager ActorRef.
+  */
+case class ResponderData(system: ActorSystem, applicationStateActor: ActorRef, responderManager: ActorRef, storeManager: ActorRef)
```

Comment:
I like this.

Reply:
should make refactoring in the future much easier :-)